core/src/components/text/text.tsx

```diff
@@ -11,7 +11,9 @@ import type { Color } from '../../interface';
  */
 @Component({
   tag: 'ion-text',
-  styleUrl: 'text.scss',
+  styleUrls: {
+    ionic: 'text.ionic.scss',
```

Comment:
This change will result in the original implementation no longer functioning. When adding the ionic specific stylesheet, we need to make sure we backwards compatible with `md` and `ios`. 

For this example it would be:
```ts
styleUrls: {
  ios: 'text.scss',
  md: 'text.scss',
  ionic: 'text.ionic.scss'
}
```

Reply:
Related to the ion-text styles for the new theme, we don't need to add a scss file. So here we just need to update to `styleUrls: {
  ios: 'text.scss',
  md: 'text.scss',
}`
With this, for the ion-text on new theme will have the global styles present on theme.